servers/lumen/app/Http/Controllers/ServiceStatementController.php

```diff
@@ -25,81 +25,44 @@ public function getOneServiceStatement($pServerSideRowUuid)
         return response()->json(ServiceStatement::find($pServerSideRowUuid));
```

Comment:
Line no: 18. variable name should be '$serviceStatementQueryResultObj' instead of '$ServiceStatementQuery'